### DISCUSSION TITLE
wild card on ACL user for group

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#6987] [Q&A]
we have a use case we create customer group dynamically 

like 
projector-group-1
projector-group-2
projector-group-3

each time we have a new projection started.  This is something very peculiar with our app. 

But Q to Strimxzi is can I wild card the group in user ACL?  like this 

      - resource:
               type: group
               name: projector-group-*
              patternType: literal
             operation: Read
           host: "*"
      - resource:
            type: group
            name: projector-group-*
            patternType: literal
           operation: Describe
           host: "*"

I tried but still application is not starting up, and KAFKA i can see AUTH error  

 User:CN=gpdstechuser is Denied Operation = Describe from host =XXXXXX  on resource = Group:LITERAL:projector-group-1 for request = XX with resourceRefCount = 1 

what am I doing wrong?

**Comment by scholzj** [comment]
AFAIK Kafka's authorizer does not support any wildcards like this. You can use only `*` as any value or the `prefix` pattern type.

**Reply by sreejesh-radhakrishnan-db** [reply]
thank you, is this right way to use PREFIX?  if I may ask please

      - resource:
          type: group
          name: projector-
          patternType: prefix
        operation: Read
        host: "*"
      - resource:
          type: group
          name: projector-
          patternType: prefix
        operation: Describe
        host: "*"

**Reply by scholzj** [reply]
I think so, yes.

**Reply by sreejesh-radhakrishnan-db** [reply]
thanks it worked.
